tools/functional-tester/etcd-tester/main.go

```diff
@@ -19,19 +19,21 @@ import (
 	"log"
 	"net/http"
 	"strings"
+	"time"
 )
 
 func main() {
 	endpointStr := flag.String("agent-endpoints", "localhost:9027", "HTTP RPC endpoints of agents. Do not specify the schema.")
-	datadir := flag.String("data-dir", "agent.etcd", "etcd data directory location on agent machine.")
 	stressKeySize := flag.Int("stress-key-size", 100, "the size of each key written into etcd.")
 	stressKeySuffixRange := flag.Int("stress-key-count", 250000, "the count of key range written into etcd.")
 	limit := flag.Int("limit", 3, "the limit of rounds to run failure set.")
+	localTest := flag.Bool("local-test", false, "'true' if run in localhost (needs TCP multi-port configuration)")
 	isV2Only := flag.Bool("v2-only", false, "'true' to run V2 only tester.")
+	waitDuration := flag.Duration("consistent-check-wait-time", 5*time.Second, "time to wait before check consistency")
```

Comment:
i do not think we should make this configurable actually. we should find a value and know about why it is a value to use instead of changing it back and forth to make the test pass.